addons/mail/static/src/messaging/component/follower_list_menu/follower_list_menu_tests.js

```diff
@@ -124,6 +125,170 @@ QUnit.test('base rendering editable', async function (assert) {
+            if (
+                route === '/web/dataset/call_kw/res.partner/read' &&
+                args.args[1][0] === 'message_follower_ids'
+            ) {
+                return [{
+                    message_follower_ids: [1, 2],
+                }];
+            }
+            return this._super(...arguments);
+        },
+    });
+    this.data['mail.followers'].records = [{
+            // simulate real return from RPC
+            // (the presence of the key and the falsy value need to be handled correctly)
+            channel_id: false,
+            id: 1,
+            name: "Mitchell Admin",
+            partner_id: this.env.session.partner_id,
+        }, {
+            // simulate real return from RPC
+            // (the presence of the key and the falsy value need to be handled correctly)
+            channel_id: false,
+            id:2,
+            name: "Eden Hazard",
+            partner_id: 11,
+        },
+    ];
+    const thread = this.env.entities.Thread.create({
+        id: 100,
+        model: 'res.partner',
+    });
+    thread.refreshFollowers();
+    await this.createFollowerListMenuComponent(thread);
+
+    assert.containsOnce(
+        document.body,
+        '.o_FollowerListMenu',
+        "should have followers menu component"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_FollowerListMenu_buttonFollowers',
+        "should have followers button"
+    );
+
+    await afterNextRender(() => {
+        document.querySelector('.o_FollowerListMenu_buttonFollowers').click();
+    });
+    assert.containsOnce(
+        document.body,
+        '.o_FollowerListMenu_dropdown',
+        "followers dropdown should be opened"
+    );
+    assert.containsN(
+        document.body,
+        '.o_Follower',
+        2,
+        "exactly two followers should be listed"
+    );
+    assert.containsN(
+        document.body,
+        '.o_Follower_name',
+        2,
+        "exactly two followers should be listed"
+    );
+    assert.strictEqual(
+        document.querySelectorAll('.o_Follower_name')[0].textContent.trim(),
+        "Mitchell Admin",
+        "first follower is 'Mitchell Admin'"
+    );
+    assert.strictEqual(
+        document.querySelectorAll('.o_Follower_name')[1].textContent.trim(),
+        "Eden Hazard",
+        "second follower is 'Eden Hazard'"
+    );
+});
+
+QUnit.test('base rendering with multiple channel followers', async function (assert) {
+    assert.expect(7);
+
+    await this.start({
+        async mockRPC(route, args) {
+            if (
+                route === '/web/dataset/call_kw/res.partner/read' &&
+                args.args[1][0] === 'message_follower_ids'
+            ) {
+                return [{
+                    message_follower_ids: [1, 2],
+                }];
+            }
+            return this._super(...arguments);
+        },
+    });
+    this.data['mail.followers'].records = [{
+            channel_id: 11,
```

Comment:
I don't like this one either, maybe just with this diff
```suggestion
    this.data['mail.followers'].records = [
        {
            channel_id: 11,
```